app/src/main/java/com/github/orkest/View/auth/SignUpForm.kt

```diff
@@ -0,0 +1,136 @@
+package com.github.orkest.View.auth
+
+import android.annotation.SuppressLint
+import android.widget.RadioGroup
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.style.TextIndent
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import com.github.orkest.Model.Providers
+import com.github.orkest.View.theme.Teal200
+import com.github.orkest.View.theme.White
+
+
+// Needs to be initialized in the main activity and called as a callback function
+// after the google authentication sign up returns
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+fun SignUpForm(navController: NavController, viewModel: AuthViewModel) {
+    Scaffold(
```

Comment:
Yes, for now it isn't, but it will probably be used to navigate to the main page of the app after the profile creation

Reply:
I think we can only navigate inside an activit,y and because we are in signUp activity maybe it will not be possible to navigate to an other activity (the main )

Reply:
Oh I see, I will then delete the authActivity and move my sign up page to the mainActivity you just created maybe. It's not the most ideal for separation of concepts but I guess there isn't another solution for now

Reply:
Thank you for you advices, I will implement all theses changes and make a commit on this pull request